MeshLib/MeshSurfaceExtraction.h

```diff
@@ -45,12 +45,20 @@ class MeshSurfaceExtraction
 	/**
 	 * Returns the 2d-element mesh representing the surface of the given mesh.
 	 * \param mesh                The original mesh
-	 * \param dir                 The direction in which face normals have to point to be considered surface elements
-	 * \param angle               The angle of the allowed deviation from the given direction (0 <= angle <= 90 degrees)
-	 * \param keepOriginalNodeIds If true, ids of mesh nodes are set to ids in original mesh, otherwise node ids are reset (as usual when creating a mesh)
-	 * \return                    A 2D mesh representing the surface in direction dir
+	 * \param dir                 The direction in which face normals have to
+	 * point to be considered surface elements
+	 * \param angle               The angle of the allowed deviation from the
+	 * given direction (0 <= angle <= 90 degrees)
+	 * \param prop_name If not empty, ids of nodes of the original subsurface
+	 * mesh are stored in a \c MeshLib::PropertyVector with the name stored in
+	 * \c prop_name
```

Comment:
And if empty original ids are dropped? If so:
* Please document that.
* Then I'd prefer a more verbose argument name, like `subsfc_nodeid_backup_prop`.
If not, then what? ^^